pstress/generate_mysqld_options.sh

```diff
@@ -0,0 +1,569 @@
+./bin/mysqld --no-defaults --help --verbose 2>/dev/null | \
+ sed '0,/Value (after reading options)/d' | \
+ egrep -v "To see what values.*is using|mysqladmin.*instead of|^[ \t]*$|\-\-\-" \
+ > ${TEMP_FILE}
+
+# mysqld options excluded from list
+# RV/HM 18.07.2017 Temporarily added to EXCLUDED_LIST: --binlog-group-commit-sync-delay due to hang issues seen in 5.7 with startup like --no-defaults --plugin-load=tokudb=ha_tokudb.so --tokudb-check-jemalloc=0 --init-file=/home/hrvoje/percona-qa/plugins_57.sql --binlog-group-commit-sync-delay=2047
+EXCLUDED_LIST=( --innodb-interpreter --innodb-interpreter-output --innodb-parallel-doublewrite-path --internal-tmp-mem-storage-engine --log-isam --log-tc --myisam-stats-method --pid-file --slow-query-log-file --terminology-use-previous --transaction-write-set-extraction --version-suffix --plugin-load --proxy-protocol-networks --utility-user-schema-access --utility-user-privileges --admin-tls-version --language --admin-port --slow-query-log-timestamp-precision --slow-query-log-use-global-control  --log-slow-filter  --disabled-storage-engines --innodb-temp-data-file-path --innodb-undo-directory --admin-address --admin-ssl-ca --admin-ssl-capath --admin-ssl-cert --admin-ssl-cipher --admin-ssl-crl --admin-ssl-crlpath --admin-ssl-key --admin-tls-ciphersuites --log-slow-verbosity --basedir --datadir --plugin-dir --lc-messages-dir --tmpdir --slave-load-tmpdir --bind-address --binlog-checksum --character-sets-dir --init-file --init-replica --init-slave --innodb-doublewrite-dir --innodb-redo-log-archive-dirs --persist-only-admin-x509-subject --replica-skip-errors --replica-type-conversions --tls-ciphersuites --utility-user-dynamic-privileges --general-log-file --log-error --innodb-data-home-dir --event-scheduler --chroot --init-slave --init-connect --debug --default-time-zone --des-key-file --ft-stopword-file --innodb-page-size --innodb-undo-tablespaces --innodb-data-file-path --innodb-ft-aux-table --innodb-ft-server-stopword-table --innodb-ft-user-stopword-table --innodb-log-arch-dir --innodb-log-group-home-dir --log-bin-index --relay-log-index --report-host --report-password --report-user --secure-file-priv --slave-skip-errors --ssl-ca --ssl-capath --ssl-cert --ssl-cipher --ssl-crl --ssl-crlpath --ssl-key --utility-user --utility-user-password --socket --socket-umask --innodb-trx-rseg-n-slots-debug --innodb-fil-make-page-dirty-debug --initialize --initialize-insecure --port --binlog-group-commit-sync-delay --innodb-directories --keyring-migration-destination --keyring-migration-host --keyring-migration-password --keyring-migration-port --keyring-migration-socket --keyring-migration-source --keyring-migration-user --mysqlx-socket --mysqlx-ssl-ca --mysqlx-bind-address --mysqlx-ssl-capath --mysqlx-ssl-cert --mysqlx-ssl-cipher --mysqlx-ssl-crl --mysqlx-ssl-crlpath --mysqlx-ssl-key --innodb-temp-tablespaces-dir --coredumper --slow-query-log-always-write-time --log-error-suppression-list)
```

Comment:
This variable does not exist --slow-query-log-timestamp-precision, but it exists in excluded list